generators/dart/text.js

```diff
@@ -274,8 +274,14 @@ Blockly.Dart['text_print'] = function(block) {
 
 Blockly.Dart['text_prompt_ext'] = function(block) {
   // Prompt function.
-  Blockly.Dart.definitions_['import_dart_html'] =
-      'import \'dart:html\' as Html;';
+  Blockly.Dart.definitions_['import_dart_io'] =
+      'import \'dart:io\';';
```

Comment:
When this runs in DartPad it generates:
```
Uncaught exception:
Unsupported operation: StdIOUtils._getStdioInputStream
```

Reply:
This is because Dart for Web does not have ```dart:io``` while the standalone Dart VM does not have ```dart:html```. However due to the fact that the rest of Blockly has no web specific function related to the web, I assumed most users would be using the standalone VM to execute their code, and if they were adding web based blocks, they would just override the prompt block's generator.

Reply:
Hmm, I'm not sure that we can assume that all of Blockly's Dart users are server-side.  Is it possible to make the code smart enough to detect what's available?

Reply:
As far as I know Dart does not support conditional imports.

Reply:
That seems to require both modules having the same API, or creating separate modules, that have the same API. ```dart:html``` and ```dart:io``` have different APIs for user input, and Blockly does not have the ability to create multiple modules.